Add maximum values to permissions reference doc (#14963)
microsoftgraph/microsoft-graph-docs: concepts/permissions-reference.md

## Microsoft accounts and work or school accounts

Not all permissions are valid for both Microsoft accounts and work or school accounts. You can check the **Microsoft Account Supported** column for each permission group to determine whether a specific permission is valid for Microsoft accounts, work or school accounts, or both.

## Limits on requested permissions per app

Azure AD limits the number of permissions that can be requested and consented by a client app. These limits depend on the `signInAudience` value for your app, shown in the app's manifest. 

| signInAudience                     | Allowed users                                            | Maximum permissions the app can request   | Maximum Microsoft Graph permissions the app can request   | Maximum permissions that can be consented in a single request         |
| ---------------------------------- | -------------------------------------------------------- | ----------------------------------------- | --------------------------------------------------------- | --------------------------------------------------------------------- |

> | AzureADMyOrg                       | Users from the organization where the app is registered  | 400                                       | 400                                                       | About 155 delegated permissions and about 300 application permissions |
> | AzureADMultipleOrgs                | Users from any Azure AD organization                     | 400                                       | 400                                                       | About 155 delegated permissions and about 300 application permissions |
> | PersonalMicrosoftAccount           | Consumer users (Outlook.com or Live.com accounts, etc.)  | 30                                        | 30                                                        | 30                                                                    |
> | AzureADandPersonalMicrosoftAccount | Consumer users and users from any Azure AD organization  | 30                                        | 30                                                        | 30                                                                    |

Reviewer: Suggestion: is there a chance of the max. limits being less than 155/300 or can we avoid the uncertainty of using "about"?
```suggestion
| AzureADMyOrg                       | Users from the organization where the app is registered  | 400                                       | 400                                                       | 155 delegated permissions and 300 application permissions |
| AzureADMultipleOrgs                | Users from any Azure AD organization                     | 400                                       | 400                                                       | 155 delegated permissions and 300 application permissions |
| PersonalMicrosoftAccount           | Consumer users (such as Outlook.com or Live.com accounts)  | 30                                        | 30                                                        | 30                                                                    |
| AzureADandPersonalMicrosoftAccount | Consumer users and users from any Azure AD organization  | 30                                        | 30                                                        | 30                                                                    |
```

Author: 155 and 300 are relatively conservative values. Usually the limit will be slightly higher. It depends on the specific permissions passed, unfortunately.
